astropy/visualization/wcsaxes/core.py

```diff
@@ -358,6 +358,34 @@ def plot_coord(self, *args, **kwargs):
 
         return super().plot(*args, **kwargs)
 
+    def text_coord(self, *args, **kwargs):
+        """
+        Print a text string using `~astropy.coordinates.SkyCoord` or
+        `~astropy.coordinates.BaseCoordinateFrame` objects onto the axes.
+
+        The first argument to
+        :meth:`~astropy.visualization.wcsaxes.WCSAxes.text_coord` should be a
+        coordinate, which will then be converted to the first two parameters to
+        `matplotlib.axes.Axes.text`. All other arguments are the same as
+        `matplotlib.axes.Axes.text`. If not specified a ``transform`` keyword
+        argument will be created based on the coordinate.
+
+        Parameters
+        ----------
+        coordinate : `~astropy.coordinates.SkyCoord` or `~astropy.coordinates.BaseCoordinateFrame`
+            The coordinate object to plot on the axes. This is converted to the
+            first two arguments to `matplotlib.axes.Axes.text`.
+
+        See Also
+        --------
+        matplotlib.axes.Axes.text :
```

Comment:
Is it more correct like this? I am not 100% sure.
```suggestion
        text : matplotlib.axes.Axes.text
```

Reply:
@pllim This is the "See Also" section, not "Parameters"/"Returns".

The trailing " :" isn't necessary here.